autotst/calculator/vibrational_analysis.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, transitionstate=None, directory=".", log_file=None):
         - log_file (str): path to the TS log file
         """
         self.ts = transitionstate
+        self.ts.get_molecules()
         self.ts.get_geometries()
```

Comment:
It may not be necessary. The `self.get_geometries` is really only need for geometry manipulation within the conformer generation so they should probably be separated. Good catch!